### ISSUE TITLE
Generation of report for android in react-native app is not working properly

### ISSUE THREAD

**Issue by gromanas** [wswebcreation/wdio-multiple-cucumber-html-reporter#27]
**Environment:**
 - Node.js version: [8.11.1]
 - NPM version: [5.6.0]
 - webdriver.io version: [e.g. 4.13.1]
 - wdio-cucumber-framework version: [2.2.8]
 - wdio-multiple-cucumber-html-reporter version: [0.3.3]

**Config of webdriver.io and the reporter**
```
reporters: ['dot', 'cucumber', 'multiple-cucumber-html'],
      reporterOptions: {
        htmlReporter: {
            jsonFolder: './test/output',
            reportFolder: './test/output/report',
            openReportInBrowser: 'yes',
            pageTitle: 'pageTitle',
            displayDuration: true,
            removeFolders: true
        }

    },
```

**Describe the bug**
I am trying to run the test suite, but in android we get the following error, and the report doesn't generated

```
/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/webdriverio/build/lib/utils/BaseReporter.js:336
                        throw _iteratorError;
                        ^

TypeError: Cannot read property 'toLowerCase' of undefined
    at eval (lodash.templateSources[1]:205:76)
    at arrayEach (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/lodash/lodash.js:516:11)
    at Function.forEach (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/lodash/lodash.js:9344:14)
    at eval (lodash.templateSources[1]:38:4)
    at _createFeaturesOverviewIndexPage (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/multiple-cucumber-html-reporter/lib/generate-report.js:449:84)
    at generateReport (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/multiple-cucumber-html-reporter/lib/generate-report.js:135:3)
    at MultipleCucumberHtmlReporter.onEnd (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/wdio-multiple-cucumber-html-reporter/build/reporter.js:160:56)
    at emitOne (events.js:116:13)
    at MultipleCucumberHtmlReporter.emit (events.js:211:7)
    at BaseReporter.handleEvent (/Users/gromanas/lotteryie/CtpIENativeApp/node_modules/webdriverio/build/lib/utils/BaseReporter.js:324:35)
```
**Android Json file**
```
[{"keyword":"Feature","description":"As a user I should be able to check Home Screen","line":1,"name":"Check Demo Screen","tags":[],"uri":"/Users/gromanas/lotteryie/CtpIENativeApp/test/features/home/home.feature","elements":[{"keyword":"Scenario","line":4,"name":"Test Demo screen","tags":[],"id":"check-demo-screen;test-demo-screen","steps":[{"arguments":[],"keyword":"Given ","name":"I wait 4 seconds","result":{"status":"passed","duration":4017000000},"line":5,"match":{"location":"can not be determined with webdriver.io"}}]}],"id":"check-demo-screen","metadata":{"browser":{},"device":"Pixel XL","platform":{"name":"android","version":"8.0"}}}]
```

**Additional context**
It doesn't work for apps.

**Comment by wswebcreation** [comment]
Thanks for this issue, I'm fixing this asap

**Comment by wswebcreation** [comment]
Fixed with version `0.4.0`
